### PR TITLE
COOPR-632: Update CLI to support partial templates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@jwang47 Please take a look

File: coopr-rest-client/src/main/java/co/cask/coopr/client/rest/RestClient.java [hunk 1]
```diff
@@ -134,6 +134,7 @@ public static void analyzeResponseCode(HttpResponse response) {
   }
 
   protected <T> List<T> getAll(URI url, Type type) throws IOException {
+    System.out.println(url.toString());
```
Comment (jwang47) [inline coopr-rest-client/src/main/java/co/cask/coopr/client/rest/RestClient.java]: remove this

**Comment by jwang47** [comment]
One minor comment, otherwise LGTM

**Comment by rpedanov** [comment]
@jwang47 done

**Comment by jwang47** [comment]
LGTM - please resolve merge conflicts

**Pull request closed without merging**